parseComCode/ReadMeParseComCloudCode.txt

```diff
@@ -4,7 +4,6 @@ As of first submission (11/24/14), the same code is in both the sandbox (deploy
 To deploy a new version:
 See https://parse.com/docs/cloud_code_guide for more.
 You can download the Parse command line tool for Windows at https://parse.com/downloads/cloud_code/cli/parse-windows/latest
-N.B. The current version of the Parse CLI (2.0.9) is unstable. If the Parse CLI is not updated, see 06/03/15 notes below.
```

Comment:
This is already obsolete. Current version is later than 2.0.9 and worked for me.